app/move/controllers/view.js

```diff
@@ -5,6 +5,7 @@ const updateSteps = require('../steps/update')
 
 const getUpdateLinks = require('./view/view.update.links')
 const getUpdateUrls = require('./view/view.update.urls')
+const setYouthMove = require('./view/view.youth-move')
```

Comment:
This stuck out as being different - so I'm wondering if there is a way to phrase this that fits the `get_Something_` pattern we're already using for these view controllers

Reply:
1. It is literally updating the move type
2. It should stick out - it's technical debt that we even have to do this

So, disagreeing ;-p ...

Reply:
I'm inclined to agree with @merlinc about this kind of sticking out a bit. And I'm not sure I agree with just because it's solving some technical debt from the API that it justifies us maybe putting it here.

If the purpose is to determine the move type correctly I wonder if we can extend the [transform method](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/146a7fbb3e24184ef0dff150f5cb97e89cbf290d/common/services/move.js#L50) in the move service to take care of this logic instead of extracting it to here?

**Edit**: After looking up the move service it actually looks like the transform method is [already doing something similar](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/146a7fbb3e24184ef0dff150f5cb97e89cbf290d/common/services/move.js#L52-L57)? So I'm wondering what extra purpose this serves?

Reply:
https://talent-intl.slack.com/archives/C010Z1KP555/p1605521639006900

So leaving this open but ignoring for now until we can resolve this properly. It's not really relevant to the events timeline PR so can safely be ignored for now